main.py

```diff
@@ -5,6 +5,9 @@
 import urllib.request
 import warnings
 
+from os import system
+import datetime as dt
```

Comment:
It is not really common way to import `datetime` like this. (It is used sometimes but mostly in the following case: `import datetime as dt; from datetime import datetime` [that I do not like, to be honest])

[Google Style Guide 2.2.4 (4)](https://google.github.io/styleguide/pyguide.html#224-decision)

Reply:
Fixed